source/stable-release_iot/feature-delta-iot.rst

```diff
@@ -0,0 +1,76 @@
+.. _feature-delta-iot:
+
+Feature Delta Offered by Celadon IoT
+####################################
+
+Following are the additional features provided by Celadon IoT.
+
+.. contents::
+    :depth: 1
+    :local:
+* 100% Google Conformance on Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) with Production ready
+* Supporting Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) on single baseline and Single Image
+* Elkhart Lake(EHL) Platform supported on Celadon with 100% compliance
+* IoT configuration support GVT-d mode with 100% conformance
+* IoT configuration support GVT-g with no conformance
+
+Real-time Clock (RTC) & Alarm Virtualization
+********************************************
+
+Celadon in Virtual Machine (CiV) project may requires operations on the RTC device, such as changes to current time and setups of alarm, to be synced to host and persistent across reboots. RTC device is purely emulated by QEMU and its state will be lost after exit of QEMU process, furthermore, guest RTC alarm is unable to wake up the guest when host is also suspended, which is the case in when using --host-pm-control option in CiV. 
```

Comment:
may require

"which is the case in when using --host-pm-control option in CiV" please check for readability

Reply:
@usivarax Yet to be fixed?

Reply:
As suggested by @sfonn modified the line before "--host-pm-control": 
RTC device is purely emulated by QEMU and its state will be lost after exit of QEMU process. Furthermore, guest RTC alarm is unable to wake up guest after host suspends.